.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
```

Comment:
Please use 4 space indentation for YAML files.  They're already nearly unreadable, I need 4 space spaces in order to comprehend them.

Reply:
Done